astro/src/content/blog/why-members-idpro.mdx

```diff
@@ -0,0 +1,66 @@
+---
+publish_date: 2024-12-16 
+title: Why We Joined IDPro
```

Comment:
I'd also consider changing the URL to fusionauth-idpro-membership-benefits

Reply:
done.